$MirOS: ports/misc/screen/patches/patch-configure_in,v 1.1 2008/03/12 23:00:44 tg Exp $
--- configure.in.orig	Fri Oct 31 13:07:52 2008
+++ configure.in	Thu Oct 15 17:25:09 2009
@@ -71,6 +71,17 @@ main()
 }
 ],AC_MSG_ERROR(Your compiler does not set the exit status - sorry))
 
+dnl from GNU screen 4.0.3 {{{
+ETCSCREENRC="\"/usr/local/etc/tscreenrc\""
+if test -n "$prefix"; then
+ETCSCREENRC="\"$prefix/etc/tscreenrc\""
+fi
+AC_MSG_CHECKING(for the global tscreenrc file)
+AC_ARG_WITH(sys-screenrc, [  --with-sys-screenrc=path  where to put the global tscreenrc file], [ ETCSCREENRC="\"${withval}\"" ])
+AC_DEFINE_UNQUOTED(ETCSCREENRC,$ETCSCREENRC)
+AC_MSG_RESULT($ETCSCREENRC)
+dnl from GNU screen 4.0.3 }}}
+
 AC_PROG_AWK
 
 AC_PROG_INSTALL
@@ -1298,6 +1309,8 @@ dnl Ptx bug workaround -- insert -lc aft
 test -n "$seqptx" && LIBS="-ltermcap -lc -lsocket -linet -lnsl -lsec -lseq"
 
 AC_TRY_RUN(main(){exit(0);},,AC_MSG_ERROR(Can't run the compiler - internal error. Sorry.))
+
+AC_CHECK_HEADERS([util.h libutil.h])
 
 AC_OUTPUT(Makefile doc/Makefile, [[
 # a hook for preserving undef directive in config.h
